.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
```

Comment:
**suggestion:** place under `[tool.isort]` section in pyproject.toml 

that should be auto-generated by seed-isort-config which is one of the precommit hook.

See geoh5py